src/external_connections/next_gen_io.py

```diff
@@ -0,0 +1,59 @@
+#Utility I/O functions for the next generation water model network.
+#The read_qlat and replace_downstreams functions are copies from nhd_io.py
+#and are placed here for now, so that this program can run without importing that module.
```

Comment:
Why do we not want to import nhd_io?

Reply:
Good point. It should be fine to import now that it is in this repo. I was originally trying to make things as standalone as much as possible.